CalibCalorimetry/HcalAlgos/src/HcalPulseContainmentAlgo.cc

```diff
@@ -80,7 +90,7 @@ std::pair<double, double> HcalPulseContainmentAlgo::calcpair(double truefc) {
   char s[80];
   sprintf (s, "%8.2f %8.4f %8.4f %8.5f %8.5f %8.5f ",
 	   truefc, tmin, tmax, integral, corfactor, recofc);
-  cout << s;
+  std::cout << s << std::endl;
```

Comment:
My understanding is that the use of std::cout is discouraged. Please use the relevant MessageLogger instead, https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger